lang/lang-de-DE.json

```diff
@@ -0,0 +1,191 @@
+{
+   "language-info":{
+      "name":"Deutsch",
+      "code":"de-DE"
+   },
+   "strings":{
+      "buttonOK":"OK",
+      "buttonCancel":"Abbrechen",
+      "buttonSave":"Speichern",
+      "menu":{
+         "file":{
+            "file":"&Datei",
+            "apply":"Profil &anwenden",
+            "applyKey":"Enter",
+            "rename":"&Umbenennen",
+            "renameKey":"F2",
+            "new":"&Neu",
+            "newKey":"Strg+N,
```

Comment:
Please add a quotation mark before the comma.
`
"newKey":"Strg+N",
`

Reply:
fixed my mistake